llvm/lib/Transforms/Instrumentation/AddressSanitizer.cpp

```diff
@@ -1323,11 +1498,21 @@ static bool isUnsupportedAMDGPUAddrspace(Value *Addr) {
 static bool isUnsupportedSPIRAccess(Value *Addr, Function *Func) {
   std::ignore = Func;
   Type *PtrTy = cast<PointerType>(Addr->getType()->getScalarType());
-  // Private address space: skip kernel arguments
-  if (PtrTy->getPointerAddressSpace() == 0) {
-    // FIXME: Currently we don't suppport all private variables
-    // return Func->getCallingConv() == CallingConv::SPIR_KERNEL &&
-    //        isa<Argument>(Addr);
+  if (PtrTy->getPointerAddressSpace() == 0) { // Private address space
```

Comment:
can hardcoded 0 be replaced with a predefined maco?

Reply:
nice catch, done